vncviewer/DesktopWindow.cxx

```diff
@@ -51,6 +51,8 @@
 
 #ifdef __APPLE__
 #include "cocoa.h"
+#include <CoreGraphics/CGGeometry.h>
+#include <CoreGraphics/CGRemoteOperation.h>
```

Comment:
The compiler can't find these here. I'm using an older SDK and gcc instead of clang. Hower `#include <Carbon/Carbon.h>` works for me. Does that work for you?

Reply:
i don't have a macOS device ^^;; i was just fiddling until i got it working on CI. what's preferable in these cases? to drop support for macOS altogether? (in any case, moved to #1212).

Reply:
It works for me with those changes, so go with that. :)